srcpkgs/libchewing-devel/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'libchewing'
+pkgname=libchewing
+version=0.5.1
+revision=1
+#wrksrc=
+#create_wrksrc=yes
+#only_for_archs="i686 x86_64"
+build_style=cmake
+#configure_args=""
+#make_build_args=""
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends="autoconf automake libtool clang gcc"
+makedepends="sqlite-devel"
+depends="sqlite"
+short_desc="An intelligent phonetic (Zhuyin/Bopomofo) input method"
+maintainer="Robert Stancil <robert.stancil@mavs.uta.edu>"
+license="BSD-3-Clause, LGPL-2.1-only"
+homepage="http://chewing.im/"
+distfiles="https://github.com/chewing/libchewing/releases/download/v${version}/${pkgname}-${version}.tar.bz2"
+checksum="9708c63415fa6034435c0f38100e7d30d0e1bac927f67bec6dfeb3fef016172b"
+
+libchewing-devel_package() {
+	depends="libchewing>=${version}_${revision}"
+	short_desc+=" - development files"
+	pkg_install() {
+		vmove usr/include
+		vmove usr/lib/pkgconfig
+		vmove "usr/lib/*.a"
+		vmove "usr/lib/*.so"
+		vmove usr/share
```

Comment:
Please make sure there is no files in /usr/share that belong in the main package and not in the -devel package, be aware that in updates that might change so it is recomended to vmove the correct paths under /usr/share